gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,226 @@
+            <h2 className="nx-h2">Description</h2>
+          </div>
+        </header>
+        <div className="nx-tile-content">
+          <NxP>
+            <NxCode>NxGlobalSidebar</NxCode> is a collapsible page level sidebar. It has two main sections:
+          </NxP>
+          <ul className="nx-list nx-list--bulleted">
+            <li className="nx-list__item">A header which contains branding as well as the open/close toggle</li>
+            <li className="nx-list__item">
+              A navigation link section which display an icon plus text in the open state and an icon in the closed
+              state
+            </li>
+          </ul>
+          <NxP>
+            Because <NxCode>NxGlobalSidebar</NxCode> contains product branding it should not be used in conjunction
+            with <NxCode>NxPageHeader</NxCode> or <NxCode>NxNexusPageHeader</NxCode>. Instead it should be paired with
+            <NxCode>NxSystemHeader</NxCode> (not yet added to the gallery).
+          </NxP>
+          <NxP>
+            Addition custom content may be added below the navigation links if desired, care should be taken that such
+            content can adapt to both the open and closed states.
+          </NxP>
+          <section className="nx-tile-subsection">
+            <header className="nx-tile-subsection__header">
+              <h3 className="nx-h3"><NxCode>NxGlobalSidebar</NxCode> Props</h3>
+            </header>
+            <NxTable>
+              <NxTable.Head>
+                <NxTable.Row>
+                  <NxTable.Cell>Prop</NxTable.Cell>
+                  <NxTable.Cell>Type</NxTable.Cell>
+                  <NxTable.Cell>Required</NxTable.Cell>
+                  <NxTable.Cell>Details</NxTable.Cell>
+                </NxTable.Row>
+              </NxTable.Head>
+              <NxTable.Body>
+                <NxTable.Row>
+                  <NxTable.Cell>isOpen</NxTable.Cell>
+                  <NxTable.Cell>Boolean</NxTable.Cell>
+                  <NxTable.Cell>No</NxTable.Cell>
+                  <NxTable.Cell></NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>toggleOpenIcon</NxTable.Cell>
+                  <NxTable.Cell>IconDefinition</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>
+                    An icon used in the sidebar's open/close toggle to represent Opening the sidebar from a closed
+                    state.
+                  </NxTable.Cell>
+                </NxTable.Row>
+                <NxTable.Row>
+                  <NxTable.Cell>toggleClosedIcon</NxTable.Cell>
+                  <NxTable.Cell>IconDefinition</NxTable.Cell>
+                  <NxTable.Cell>Yes</NxTable.Cell>
+                  <NxTable.Cell>
+                    An icon used in the sidebar's open/close toggle to represent Opening the sidebar from an open
+                    state.
```

Comment:
I find the descriptions on these two confusing

Reply:
> I find the descriptions on these two confusing

Hopefully this is clearer: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/190afc675e47acd0ce72dab365dd5d16e035f5ec